site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -190,6 +190,12 @@ The JSON payload that is sent is a `FleetAutoscaleReview` data structure and a `
 The `FleetAutoscaleResponse`'s `Replica` field is used to set the target `Fleet` count with each sync interval, thereby
 providing the autoscaling functionality.
 
+{{< beta title="Fleet Metadata in Webhook Requests" gate="FleetAutoscaleRequestMetaData" >}}
```

Comment:
Forgot to mention, will need to be feature short coded like so:

https://agones.dev/site/docs/contribute/documentation-editing-contribution/#within-a-page 

1.54.0 just released, so the shortcode and version numbers in this PR will also need to be updated.

Thanks!